### PR TITLE
Add -TurnOnAutoDeleteDisk and -TurnOffAutoDelete to Set-GceInstance

### PR DESCRIPTION
**Pull request by quoctruong**
These parameters are used to turn on/off autodelete in the disks of the instance.

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-powershell/416)

### CODE REVIEW

File: Google.PowerShell/Compute/GceInstanceCmdlets.cs [hunk 1]
```diff
@@ -1223,37 +1296,50 @@ private void ProcessAccessConfig()
             }
         }
 
+        // Given a list of string, transform any selflink URI into attached disk name.
+        private IEnumerable<string> GetAttachedDiskName(string[] diskNames)
+        {
+            if (diskNames.Any(diskName => Uri.IsWellFormedUriString(diskName, UriKind.Absolute)))
+            {
+                // The disks on the GCE instance we need to set.
+                // This is used for the case when there are selflinks instead of disk names.
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceInstanceCmdlets.cs]: nit: selflinks -> self links

File: Google.PowerShell/Compute/GceInstanceCmdlets.cs [hunk 2]
```diff
@@ -1285,6 +1371,34 @@ private void ProcessDisk()
         }
 
         /// <summary>
+        /// ProcessRecord for AutoDeleteDisk parameter set.
+        /// </summary>
+        private void ProcessAutoDeleteDisk()
+        {
+            foreach (string diskName in GetAttachedDiskName(TurnOnAutoDeleteDisk))
```
Comment (iantalarico) [inline Google.PowerShell/Compute/GceInstanceCmdlets.cs]: Might be nice to make a helper here.  This is nearly duplicate code.

**Comment by quoctruong** [comment]
@iantalarico PTAL

**Comment by quoctruong** [comment]
Thanks!

**Pull request merged**